docs/articles/systemd.md

```diff
@@ -85,6 +85,17 @@ setting `OPTIONS`:
 You can also set other environment variables in this file, for example, the
 `HTTP_PROXY` environment variables described below.
 
+If you want to modify the `ExecStart` configuration, make sure you clear it
```

Comment:
Tibor's comment is valid.  This would make the why more explicit:

-->https://gist.github.com/moxiegirl/fbf29cb3dccf577b336b

To modify the `ExecStart` configuration, specify an empty configuration followed by a new configuration as follows:

    [Service]
    ExecStart=
    ExecStart=/usr/bin/docker -d -H fd:// --bip=172.17.42.1/16

If you fail to specify an empty configuration, Docker reports an error such as:

    docker.service has more than one ExecStart= setting, which is only allowed for Type=oneshot services. Refusing.